.github/workflows/main.yml

```diff
@@ -0,0 +1,56 @@
+name: Release to dockerhub
+on:
+  push:
+    tags:
+      - dockerhub_release_*  # Launch dockerhub push on dockerhub_release_* tags
```

Comment:
Nitpick: I wonder if we should use a regular tag here, as opposed to a dockerhub_release tag specific.

Reply:
whatever you think would be the most useful

Reply:
How about any tag?

Reply:
I'm not very familiar with github actions. Does this line mean that the action will only be triggered when a `dockerhub_release_` tag is created?

Reply:
That's my understanding.